graphql_compiler/tests/test_data_tools/schema.sql

```diff
@@ -0,0 +1,80 @@
+### UUIDs ###
+CREATE CLASS UniquelyIdentifiable
+CREATE PROPERTY UniquelyIdentifiable.uuid String
+# make uuid default to '' and insert such a record on purpose
+# this will ensure that no UniquelyIdentifiable is added without a valid uuid field
+ALTER PROPERTY UniquelyIdentifiable.uuid DEFAULT ''
+INSERT INTO UniquelyIdentifiable SET uuid = ''
+CREATE INDEX UniquelyIdentifiable.uuid UNIQUE_HASH_INDEX
+###############
+
+### Entity ###
+CREATE CLASS Entity EXTENDS V, UniquelyIdentifiable ABSTRACT
+CREATE PROPERTY Entity.name String
+CREATE INDEX Entity.name NOTUNIQUE
```

Comment:
Did you want names to be unique? You seem to imply it in `animals.py` on line 55. If so, make this a unique index.